Talk Proposals/Stop Worrying.md

```diff
@@ -0,0 +1,40 @@
+
+## Description
+
+New programming languages and frameworks are great, but they don't address the fundamental problems we tend to experience on every project: long hours, late releases, and unhappy customers. We're told that agile development practices help with these problems, but getting started can be a catch-22. For example, agile suggests you should test your code, but to do so it has to be well-designed, but to change the design it needs to be tested. How do you get on the ride?
+
+I'd like to tell you the story of how I got on the ride — how I made it through a series of worries one at a time. Each worry came from a real need, and identifying that need helped me find an agile development process to meet it. Another worry would crop up afterward, but I just repeated the process of following the worry again. Now my day-to-day development involves a lot less stress and a lot more fun!
+
+Here's the series of worries I went through:
+
+- **"Will we get the code done in time?"** (The need here is for predictability.)
+- **"Will I break something?"** (The need is reliability.)
+- **"Am I wasting time writing tests?"** (Testability.)
+- **"Can my code handle future changes?"** (Flexibility.)
+
+For each of these worries, you'll learn:
+
+- Why the need underlying the worry is important to meet.
+- Which agile development practice addresses that need.
+- What that development practice looks like in a simple scenario.
+- Where you can go to dig more deeply into learning that development practice.
+
+You'll walk away being able to identify what your biggest need is right now and what steps you can take to begin to resolve it.
+
+This session won't remove all your worry: it isn't counseling! But these practices are likely to make a significant change in your quality of developer life. You may even find that you fall in love with development all over again.
+
+## Submission Note
+
+A great way to get into agile development is to apply one practice at a time, based on which will provide the most benefit first. This sequence of practices worked well for me--in fact, some of the later practices weren't even comprehensible until applying earlier practices surfaced the need for them. My hope is to help people who are uninterested in agile, or who are overwhelmed by it, or who are starting with the wrong practices, to get started with a practice that will meet them where they are.
+
+The agile development practices I'll be presenting are:
+
+- The need for predictability can be met by delivering **user stories**, small slices of customer-visible functionality. Working a slice at a time rather than a layer at a time gives you a better sense of how long features will really take to complete.
+- The need for reliability can be met by writing **automated tests** to regression-test your code. We'll look briefly at both end-to-end tests and unit tests.
```

Comment:
🤔  I suppose you're not saying testing is "agile" as much as "the need for reliability" is. Does that sound right?